.npmignore

```diff
@@ -9,5 +9,5 @@
 !/README.md
 !/VERSION
 !/package.json
-!/lib/**/*
```

Comment:
+ esm?

Reply:
Updated! My original thought process was that since we're defining both entrypoints yet we should just expose cjs/, but it would be weird to have things referencing `esm/` in the package but not include it.